src/hooks/cgroups/pbs_cgroups.PY

```diff
@@ -4156,8 +4150,6 @@ class CgroupUtils(object):
                 raise CgroupLimitError('Invalid limit value: %s' % (value))
             else:
                 raise
-        except KeyboardInterrupt:
-            raise
         except:
             raise
```

Comment:
There is no real purpose to an except: raise.  If you don't have this statement, all uncaught exceptions will be just that, uncaught.  This just makes you catch them and reraise them.

Reply:
Fixed.